examples/data/e2e_tests/data/exports/datasets/mappingConformanceRule.json

```diff
@@ -0,0 +1,31 @@
+{
+  "metadata": {
+    "exportVersion": 1
+  },
+  "item": {
+    "name": "mappingConformanceRule",
+    "description": "",
+    "hdfsPath": "/std/conformanceRulesData",
+    "hdfsPublishPath": "/publish/mappingConformanceRule",
+    "schemaName": "conformanceRulesData",
+    "schemaVersion": 1,
+    "conformance": [
+      {
+        "_t": "MappingConformanceRule",
```

Comment:
Would it be possible to have multiple mapping rules here, or it doesn't make sense?

Reply:
For one of the next versions, I'll prepare a more complex test of the mapping table rule.